demonstrations/tutorial_variational_classifier.py

```diff
@@ -234,22 +233,23 @@ def cost(var, X, Y):
 # variational classifier and turn them into predictions in
 # :math:`\{-1,1\}` by taking the sign of the output.
 
-var = var_init
+weights = weights_init
+bias = bias_init
 for it in range(25):
 
     # Update the weights by one optimizer step
     batch_index = np.random.randint(0, len(X), (batch_size,))
     X_batch = X[batch_index]
     Y_batch = Y[batch_index]
-    var = opt.step(lambda v: cost(v, X_batch, Y_batch), var)
+    weights, bias = opt.step(lambda w, b: cost(w, b, X_batch, Y_batch), weights, bias)
```

Comment:
```suggestion
    weights, bias, _ = opt.step(cost, weights, bias, X_batch, Y_batch)
```

Don't need to use a lambda function anymore either.